brickflow_plugins/databricks/uc_to_snowflake_operator.py

```diff
@@ -376,17 +377,22 @@ def get_sf_poststeps(self):
         return queries
 
     def get_sf_postgrants(self):
-        post_grantee_role = (
-            self.parameters["sf_grantee_roles"]
+        post_grantee_roles = (
+            self.parameters["sf_grantee_roles"].split(',')
```

Comment:
if it's going to be comma separated, may be add a trim so we could avoid spaces.

Reply:
Thanks for the feedback, there was a strip() below but I have moved it up here for readability.